tests/sqlparser_postgres.rs

```diff
@@ -1432,6 +1433,29 @@ fn parse_execute() {
                 Expr::Value(number("1")),
                 Expr::Value(Value::SingleQuotedString("t".to_string()))
             ],
+            using: vec![]
+        }
+    );
+
+    let stmt = pg_and_generic()
+        .verified_stmt("EXECUTE a USING CAST(1337 AS SMALLINT), CAST(7331 AS SMALLINT)");
+    assert_eq!(
+        stmt,
+        Statement::Execute {
+            name: "a".into(),
+            parameters: vec![],
+            using: vec![
+                Expr::Cast {
+                    expr: Box::new(Expr::Value(Value::Number("1337".parse().unwrap(), false))),
```

Comment:
😆

Reply:
:slightly_smiling_face: No problem, and thank you @alamb !